.travis.yml

```diff
@@ -22,7 +22,7 @@ git:
   depth: 10000
 
 install:
-  - export IRIS_TEST_DATA_REF="942e6344d6e6d00c85d824dadd49b5d6f57f4b4a"
+  - export IRIS_TEST_DATA_REF="7c0e32c8812b464e467a9555bdc25dc1e0c5be0c"
```

Comment:
> Did we really need to add this new dataset?

we already have, so this is a little out of scope.  I am keen to have access to this data set for testing, and this test may be the first of a number.  Plotting these data sets has been a particular challenge for some of our potential users